### DISCUSSION TITLE
A little update about me

### DISCUSSION THREAD

**Discussion by Nickwasused** [twireapp/Twire#407] [General]
I stared working on Twire sometime around April 2021 with some simple fixes and features. My last fix submitted was on February 23, 2022, since then I have not worked more than 2 hours on the project. The reasons behind this are:

- Me having less time in general
- Not having enough experience and knowledge about Android App Development. Most of the features implemented were just some addition to an already working features or simple fixes, e.g. https://github.com/twireapp/Twire/pull/247 or https://github.com/twireapp/Twire/pull/232
- No motivation to learn Android App development

### Looking Forward
I hope that new People will come to work on this Project and that Twitch will not do some weird stuff. (Yes, I am looking at you 6 out of 6 ads.)

### PR 217 (Proxy)
As of now, I am going to close my last open PR (#217). Somebody with more knowledge could probably implement this in way less time and better than I have. My personal requirement for implementing a proxy was that the server code is open-source. (The Code was based on TTV.LOL and they wanted to make the code public since April 18, 2021 (https://github.com/TTV-LOL/extensions/issues/1#issuecomment-821912086)).

**Comment by samfundev** [comment]
Thanks for all your work you've done! It means a lot to me that you would help me with maintaining Twire. I wish you the best in whatever you do next. Don't worry about your open PR, I'll try to finish up the work that you've started.
